cvat/schema.yml

```diff
@@ -385,7 +261,7 @@ paths:
       - in: query
         name: org_id
         schema:
-          type: integer
+          type: string
```

Comment:
This doesn't seem right. `org_id` should be an integer.

Reply:
Fixed